src/main/web/templates/handlebars/partials/json-ld/homepage.handlebars

```diff
@@ -0,0 +1,8 @@
+"@type": "Organization",
```

Comment:
Out of scope but this should prob be: `GovernmentOrganization` https://schema.org/GovernmentOrganization

Reply:
👍